src/Api/PubnubApi/Pubnub.cs

```diff
@@ -19,7 +19,7 @@ public class Pubnub
 
         private readonly string instanceId;
 
-        private static string sdkVersion = string.Format("{0}CSharp4.2.0.0", PNPlatform.Get());
+        private static string sdkVersion = string.Format("{0}CSharp4.3.0.0", PNPlatform.Get());
```

Comment:
Would be great if we could pull this from somewhere in the future.

Reply:
![image](https://user-images.githubusercontent.com/2434451/70643133-7d246300-1c66-11ea-9ead-050500baa758.png)
SDK has AssemblyInfo.cs files which needs manual update. AssemblyInfo.cs updates version numbers on the individual Pubnub.dll properties.